deploy/images/controller/Dockerfile

```diff
@@ -1,17 +1,24 @@
 # Use distroless image which already includes ca-certificates
-FROM gcr.io/distroless/static:nonroot
+FROM ubuntu:latest
```

Comment:
We should look into if we really need ubuntu or if we can get away with a smaller image